libraries/provision/generate_pools_json_from_aws.py

```diff
@@ -92,6 +89,13 @@ def get_public_dns_names_cloudformation_stack(stackname):
     return get_public_dns_names(instances_for_stack)
 
 
+def get_private_dns_names_cloudformation_stack(stackname):
+
+    instances_for_stack = get_running_instances_for_cloudformation_stack(stackname)
```

Comment:
I think it will in practice, but there's no real guarantee that I know of .. we'd have to dig into the Cloudformation API docs.  

Having said that, it's not important for our current use.  This was added to be slightly better than the current situation, which it is.